test/TestAsset/test/test.js

```diff
@@ -1,3 +1,5 @@
 {
-  "test": "twig"
+    'test';
+:
+    'twig';
 }
```

Comment:
this looks wrong

Reply:
this should have been json not js, hence why it was invalidly formatted. corrected to the original syntax, and changed the file ext to json

Reply:
it was looking for `.js` tho:
```
Twig\Error\LoaderError: Unable to find template "@test/test.js" (looked into: /github/workspace/test/TestAsset/test).
```